### PR TITLE
implement automapper

### PR DESCRIPTION
**Pull request by abhijeetkumarLnG**

### CODE REVIEW

File: Pspcl/Pspcl.Core/Domain/Stock.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 namespace Pspcl.Core.Domain
 {
-    public class Stock
+    public class Entity
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Core/Domain/Stock.cs]: What is the need of doing this ?

File: Pspcl/Pspcl.Web/Controllers/AddStockController.cs [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
+using Microsoft.EntityFrameworkCore.Metadata.Internal;
+using Pspcl.DBConnect;
+using Pspcl.Services;
+using Pspcl.Web.Models;
 
 namespace Pspcl.Web.Controllers
 {
-    public class AddStockController : Controller
+    
+    public class AddStockController: Controller
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Web/Controllers/AddStockController.cs]: Check naming convention.. It should not be AddStockController

Comment (abhijeetkumarLnG) [reply Pspcl/Pspcl.Web/Controllers/AddStockController.cs]: changed it to stockView controller

File: Pspcl/Pspcl.Core/Domain/Stock.cs (hunk 1 above)
Comment (abhijeetkumarLnG) [reply Pspcl/Pspcl.Core/Domain/Stock.cs]: it is happend due to property name on view model and database entity are not matching so automapper used this default naming convention to map the properties

File: Pspcl/Pspcl.Web/Controllers/AddStockController.cs [hunk 3]
```diff
@@ -1,12 +1,25 @@
...
     {
-        public IActionResult AddStock()
+        private readonly StockMappingService _stockMappingService;
+        public AddStockController(StockMappingService stockMappingService)
         {
-            return View();
+            _stockMappingService = stockMappingService;
+        }
+        public IActionResult AddStock(Model model)
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Web/Controllers/AddStockController.cs]: If this is a get action, why we are passing model as parameter, please clear your concepts about action and their types

File: Pspcl/Pspcl.Core/Domain/Stock.cs (hunk 1 above)
Comment (sandeepkumarlng) [reply Pspcl/Pspcl.Core/Domain/Stock.cs]: So, you gona change it to Entity for all entity classes ?

Comment (abhijeetkumarLnG) [reply Pspcl/Pspcl.Core/Domain/Stock.cs]: i changed the entity to stock

**Review by sandeepkumarlng — APPROVED** [review]

**Pull request merged**